flink-runtime/src/main/java/org/apache/flink/runtime/util/EnvironmentInformation.java

```diff
@@ -469,7 +469,9 @@ public static void logEnvironmentInfo(
                 }
             }
 
-            log.info(" Classpath: " + System.getProperty("java.class.path"));
+            String classPath = System.getProperty("java.class.path");
+
+            log.info(" Classpath: " + classPath.replace("/*", "/ *"));
```

Comment:
The fix only works for this single case. Do we have a more general way to fix it? For example, change the way that the web UI displays '/*'?

Reply:
I can modify it in the web UI display. The reason for only modifying here is that the global modification also restricts developers from printing logs with comments. Do I need to modify the web UI next?

Reply:
Sorry for the late reply. Could you figure out if that is a big change or a small change? I guess there is no much need to highlight comments in webui. If this is small change, I think we can do that directly.

Reply:
Any update? If there is no simple way to solve the issue at the UI side, let's not do that.
For this single case, I think replacing "/*" with "/ *" is not a good solution because it changes the path itself. I am not sure if surrounding the classpath string with '"' solves the problem. If so, I think that will be a better solution. What do you think?